packages/validator/src/metrics.ts

```diff
@@ -82,25 +82,25 @@ export function getMetrics(register: MetricsRegisterExtra, gitData: LodestarGitD
 
     syncCommitteeStepCallProduceMessage: register.histogram({
       name: "vc_sync_committee_step_call_produce_message_seconds",
-      help: "Time between 1/3 of slot and call produce message",
-      // Max wait time is 1 / 3 of slot
+      help: "Time between SYNC_MESSAGE_DUE_BPS of slot and call produce message",
+      // Max wait time is 4s of slot
```

Comment:
same here, probably wanna do 
```suggestion
      // Max wait time is SYNC_MESSAGE_DUE_BPS of slot
```